z2/src/setup.rs

```diff
@@ -218,6 +219,9 @@ impl Setup {
                     blocks_per_hour: 3600,
                     rewards_per_hour: 51_000_000_000_000_000_000_000u128.into(),
                 },
+                block_request_limit: block_request_limit_default(),
+                max_blocks_in_flight: max_blocks_in_flight_default(),
+                block_request_batch_size: block_request_batch_size_default(),
```

Comment:
Could we have a brief docs/ file on the algorithm you intended to use here, please? I can kind of work it out by looking at the patch, but the patch is in several files - a quick one-liner that just points to the right comments might work out well?

Reply:
Will add a doc-comment to `BlockStore` if you don't disagree - I think all of the relevant code is here and that way we keep the algorithm description close to the implementation?

Reply:
Sounds good to me - thanks!